server/src/utils/email.utils.ts

```diff
@@ -0,0 +1,78 @@
+import nodemailer from 'nodemailer';
+import path from 'path';
+import ejs from 'ejs';
+import dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
ya i think if u import dotenv in `app.ts` or wherever the root is or when app.use is called, u don't need to import it everywhere

Reply:
okk